### PR TITLE
EM-360: Update dropdwon to use region List from db

### PR DESCRIPTION
**Pull request by w8896699**

### CODE REVIEW

File: src/app/projects/add-edit-project/add-edit-project.component.ts [hunk 1]
```diff
@@ -170,6 +160,17 @@ export class AddEditProjectComponent implements OnInit {
...
+    this.config.lists.map(item => {
+      switch (item.type) {
+        case 'region':
+          this.temp.push(Object.assign({}, item));
+          break;
+      }
+    });
+    for(var _i = 0; _i<this.temp.length; _i++){
+      this.regions.push(this.temp[_i].name);
+    }
```
Comment (marklise) [inline src/app/projects/add-edit-project/add-edit-project.component.ts]: This is an unecessary double loop.  Minimize to one.

Comment (w8896699) [reply src/app/projects/add-edit-project/add-edit-project.component.ts]: Hi Mark, I think I accidentally add Kathy's commit into this pr. I am still not really comfortable with git commit process. I will ask Max about what is going on here tomorrow. Sorry for create a mass here..

**Comment by w8896699** [comment]
I rebased this and it only have one commit now.

**Comment by kathydo** [comment]
QA and code reviewed by Kathy. Region in drop downs, and project details displays as expected. The region also correctly shows up in project edit page

**Pull request merged**